etc/fetchers/fetch-faq-sheet.js

```diff
@@ -0,0 +1,30 @@
+const cheerio = require("cheerio");
+const fs = require("fs");
+const path = require("path");
+const fetch = require("cross-fetch");
+const FAQ_LINK = "https://kcov.id/wbw-faq";
+
+module.exports.fetchFaqSheet = async function fetchFaqSheet() {
```

Comment:
```suggestion
module.exports.fetchFaqSheets = async function fetchFaqSheets() {
```

For consistency, please always use the plural form of `Sheets` everywhere, including the name of this file.
Please rename the file into `fetch-faq-sheets.js`.

Reply:
oke mas @zainfathoni. will rename it soon